src/com/oltpbenchmark/WorkloadConfiguration.java

```diff
@@ -43,14 +43,16 @@ public void setBenchmarkName(String benchmarkName) {
         this.benchmarkName = benchmarkName;
     }
 
-    private List<String> dbConnections;
+    private List<String> nodes;
 	private String db_name;
```

Comment:
Nit: indentation seems inconsistent in this file.

Reply:
Reformatted the entire code...